client/src/modules/stock/exit/modals/findDepot.modal.js

```diff
@@ -17,7 +17,7 @@ function StockFindDepotModalController(Instance, Depot, Notify, Data, Stock) {
   vm.submit = submit;
   vm.cancel = cancel;
 
-  Depot.read()
+  Depot.read(null, { only_user : true })
```

Comment:
My fear is that in the context where the health zones will use BHIMA to manage stocks, the BCZs would risk having the possibility of having the possibility of distributing products (By mistake) to the depots that do not concern them

Reply:
This doesn't really make sense though.  Because then each CDR will need to have access to every BCZ depot.  Are you sure that makes sense?

Additionally, to send a requisition to the BCZ, the CS will need to have access to change the stock in the BCZ...